packages/xstate-inspect/src/serialize.ts

```diff
@@ -20,13 +20,16 @@ export function selectivelyStringify<T extends object>(
   });
 }
 
-export function stringifyState(state: AnyState, replacer?: Replacer): string {
+export function stringifyState(
+  { machine, configuration, ...state }: AnyState,
+  replacer?: Replacer
+): string {
   return selectivelyStringify(state, ['context', 'event', '_event'], replacer);
 }
 
 export function stringifyMachine(
   machine: AnyStateMachine,
   replacer?: Replacer
 ): string {
-  return selectivelyStringify(machine, ['context'], replacer);
+  return selectivelyStringify(machine.definition, ['context'], replacer);
```

Comment:
I basically limit what is being stringified here as this should be everything that has to be potentially replaced. Previously we were passing the whole `machine` and it referenced context in a couple of properties other than `context` (such as `_context`, `config` and maybe something else too)